docs/mongocli/command/mongocli-atlas-customDbRoles-list.txt

```diff
@@ -40,18 +40,10 @@ Options
      - 
      - false
      - help for list
-   * - --limit
-     - int
-     - false
-     - Number of items per results page.
```

Comment:
This doc change is introduced because the list options only existed on the CLI side, not the API